src/test/java/redis/clients/jedis/tests/commands/AllKindOfValuesCommandsTest.java

```diff
@@ -210,16 +210,6 @@ public void rename() {
   }
 
   @Test
-  public void renameOldAndNewAreTheSame() {
```

Comment:
@HeartSaVioR my bad. I wasn't running the latest unstable version of redis in my computer and the `rename` option changed a couple of days ago. 

I'll rollback this test.